### PR TITLE
[PT-BR] Translate `osu!_wiki`

### PR DESCRIPTION
**Pull request by GranularBow8193**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Comment by Walavouchey** [comment]
thank you, but in the future, read the [best practices page](https://osu.ppy.sh/wiki/en/osu%21_wiki/Contribution_guide/Best_practices#opening-a-pull-request) on the contribution guide. in particular:

- make a new branch before committing changes. don't pr from your `master` branch
- put a descriptive commit name. leaving it at default makes it difficult to tell what it achieves
- put a descriptive title on your pr

other than that, there doesn't seem to be any formatting errors. if you know someone who can review this, tell them to check it and drop a few comments here. the pr will be merged once someone does or if about 2 weeks pass without activity

File: wiki/osu!_wiki/pt-br.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# osu! wiki
+
+O **osu! wiki** é uma base de dados de código aberto escrita e mantida por [colaboradores voluntários](https://github.com/ppy/osu-wiki/graphs/contributors). Ela contém uma vasta gama de informação em tópicos relacionados ao osu!, sua comunidade, e atividades e conceitos relacionados. O wiki serve como uma fonte oficial de conhecimento e é considerada como uma documentação da história do osu!.
+
+Todas as discussões que abrangem o wiki em geral, seu desenvolvimento, e mudanças dos artigos, acontece no canal `#osu-wiki` do servidor oficial do [osu!dev Discord server](/wiki/Community/osu!dev_Discord_server). Para aprender com você pode ajudar o osu! wiki, consulte os seguinte artigos:
```
Comment (LeoFLT) [inline wiki/osu!_wiki/pt-br.md:5]:
```suggestion
Todas as discussões que abrangem o wiki em geral, seu desenvolvimento, e mudanças dos artigos, acontece no canal `#osu-wiki` do [servidor oficial do osu!dev no Discord](/wiki/Community/osu!dev_Discord_server). Para aprender com você pode ajudar o osu! wiki, consulte os seguinte artigos:
```

File: wiki/osu!_wiki/pt-br.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+Traduções em diferentes línguas estão disponíveis para alguns artigos. Elas são feitas pelos próprios membros da comunidade, e a escolha da língua é apenas limitada pela [lista de línguas](/wiki/Article_styling_criteria/Formatting#locales) suportada pelo próprio site.
+
+## Modelo
+
+O osu! wiki é construído  em torno de um conceito de acessibilidade: edições e traduções são encorajadas e podem ser feitas por qualquer pessoa capaz e conhecedora. Os dados dos artigos estão armazenados em um [repositório GitHub](https://github.com/ppy/osu-wiki) e atualizados pelos colaboradores através do [uso de pedidos extração](/wiki/osu!_wiki/Contribution_guide), que são posteriormente revistos e fundidos pelos [conservadores da wiki](/wiki/osu!_wiki/Maintenance/List_of_maintainers).
```
Comment (LeoFLT) [inline wiki/osu!_wiki/pt-br.md:26]:
```suggestion
O osu! wiki é construído em torno de um conceito de acessibilidade: edições e traduções são encorajadas e podem ser feitas por qualquer pessoa capaz e conhecedora. Os dados dos artigos estão armazenados em um [repositório GitHub](https://github.com/ppy/osu-wiki) e atualizados pelos colaboradores através do [uso de pedidos extração](/wiki/osu!_wiki/Contribution_guide), que são posteriormente revistos e fundidos pelos [conservadores da wiki](/wiki/osu!_wiki/Maintenance/List_of_maintainers).
```

File: wiki/osu!_wiki/pt-br.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+Os artigos são compostos de acordo com o [critério de estilo](/wiki/Article_styling_criteria) do osu! wiki, o que significa que são escritos em linguagem neutra e seguem as regras de formatação e desenho. Alguns artigos informativos servem como lista de perguntas frequentes (FAQ) sobre o tema e utilizam linguagem informal. As traduções seguem o princípio da paridade do conteúdo e obedecem às mesmas regras de formatação.
+
+Apesar de ter o sufixo *wiki* em seu nome, o osu! wiki não satisfaz plenamente [o conceito Wiki](https://en.wikipedia.org/wiki/Wiki):
+
+- A melhoria de artigos existentes e a criação de novos são incentivados, mas o conhecimento básico do [GitHub](https://github.com) é necessário a fim de contribuir, o que aumenta o limiar necessário para a entrada. O osu! wiki fornece, no entanto, um [guia de contribuição](/wiki/osu!_wiki/Contribution_guide) para ajudar novos contribuidores a começar.
```
Comment (LeoFLT) [inline wiki/osu!_wiki/pt-br.md:34]:
```
aumenta o limiar necessário para a entrada
```
Limiar me parece ser uma escolha desnecessáriamente complicada aqui, por que não utilizar "aumenta a barreira de entrada" ou similares?

**Review by LeoFLT — CHANGES_REQUESTED** [review]

**Review by Walavouchey — APPROVED** [review]

**Review by LeoFLT — APPROVED** [review]
lgtm

**Pull request merged**